src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -176,7 +180,11 @@ export const MappingProfilesTransformationsModal = ({
           hidden={!isFilterPaneOpen}
         >
           <SearchForm
-            initialValues={initialSearchFormValues}
+            initialValues={{
```

Comment:
Why changes are needed there?

Reply:
For initilizing SearchForm with disabled record types values for Edit/Duplicate mapping profile pages.